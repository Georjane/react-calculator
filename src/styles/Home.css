.Home {
  background-color: rgba(231, 228, 216, 0.082);
  height: 80vh;
  padding: 1px 15px;
  font-size: larger;
}
