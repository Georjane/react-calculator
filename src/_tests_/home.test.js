import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders home page', () => {
  const tree = renderer.create(<Home />);
  expect(tree).toMatchSnapshot();
});
