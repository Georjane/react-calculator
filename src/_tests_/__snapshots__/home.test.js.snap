// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page 1`] = `
<div
  className="Home"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor et dolore magna aliqua. Ut
    <br />
    <br />
    enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip is aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. cat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>
</div>
`;
